ext/zend_test/test_arginfo.h

```diff
@@ -124,7 +124,12 @@ ZEND_END_ARG_INFO()
 
 #define arginfo_class_ZendTestForbidDynamicCall_callStatic arginfo_zend_test_void_return
 
-#define arginfo_class_ZendTestNS_Foo_method arginfo_zend_test_void_return
+#if (PHP_VERSION_ID >= 80100)
```

Comment:
you will need `php_version.h` to be included in test.c

Reply:
CI fixed (adding proper return)

Reply:
Yes, you are right, the missing return was the problem!